lib/hanami/model/migrator/mysql_adapter.rb

```diff
@@ -1,70 +1,88 @@
 module Hanami
   module Model
-    module Migrator
+    class Migrator
       # MySQL adapter
       #
       # @since 0.4.0
       # @api private
       class MySQLAdapter < Adapter
+        # @since x.x.x
+        # @api private
+        PASSWORD = 'MYSQL_PWD'.freeze
+
         # @since 0.4.0
         # @api private
         def create
-          new_connection(global: true).run %(CREATE DATABASE #{ database };)
+          new_connection(global: true).run %(CREATE DATABASE #{database};)
         rescue Sequel::DatabaseError => e
-          message = if e.message.match(/database exists/)
-            "Database creation failed. There is 1 other session using the database"
-          else
-            e.message
-          end
+          message = if e.message.match(/database exists/) # rubocop:disable Performance/RedundantMatch
+                      'Database creation failed. There is 1 other session using the database'
```

Comment:
Can we update this message? I don't think it's necessarily accurate that `There is 1 other session using the database`. https://github.com/hanami/model/pull/325/files